src/components/Board/components/Column/index.spec.js

```diff
@@ -62,6 +62,47 @@ describe('<Column />', () => {
       expect(renderCard).toHaveBeenCalledWith(column, expect.objectContaining({ id: 1, title: 'Card title 1' }), false)
     })
   })
+
+  describe('about the card adding', () => {
+    const onCardNew = jest.fn()
+
+    describe('when the component does not receive the "allowAddCard" prop', () => {
+      beforeEach(() => mount({ onCardNew }))
+
+      it('does not show the add card button', () => {
+        expect(subject.queryByText('+')).not.toBeInTheDocument()
+      })
+    })
+
+    describe('when the component receives the "allowAddCard" prop', () => {
+      beforeEach(() => mount({ onCardNew, allowAddCard: true }))
+
+      it('shows the add card button', () => {
+        expect(subject.queryByText('+')).toBeVisible()
+      })
+
+      describe('when the user clicks on the add card button', () => {
```

Comment:
Don't we need to do the inverse context? `when the user clicks on the cancel button`.

Reply:
We're testing this behavior on CardAdder component